assets/js/stripe-payment-request.js

```diff
@@ -424,6 +424,21 @@ jQuery( function( $ ) {
 			} );
 		},
 
+		debounce: function ( wait, func, immediate ) {
```

Comment:
Would it help to comment this function? The meaning of "debounce" may not be immediately obvious to someone.

Reply:
Added in 0a9b2f6